mods/ra/maps/fort-lonestar/fort-lonestar.lua

```diff
@@ -12,90 +6,62 @@ OilDerricks = { OilDerrick1, OilDerrick2, OilDerrick3, OilDerrick4 }
 SpawnPoints = { Spawn1, Spawn2, Spawn3, Spawn4 }
 Snipers = { Sniper1, Sniper2, Sniper3, Sniper4, Sniper5, Sniper6, Sniper7, Sniper8, Sniper9, Sniper10, Sniper11, Sniper12 }
 
+if Map.Difficulty == "1 Player" then
```

Comment:
I thought when people want to play on an easier level they can just choose a different "Player-difficulty" (e.g. 3 players playing with the difficulty "2 Players")
instead of using easy/normal/hard and extra scaling those with the player count.